Source/DafnyCore/AST/Modules/LiteralModuleDecl.cs

```diff
@@ -78,4 +79,119 @@ public LiteralModuleDecl(ModuleDefinition module, ModuleDefinition parent)
 
     return true;
   }
+
+  public void Resolve(Resolver resolver, Program prog, int beforeModuleResolutionErrorCount) {
+    // The declaration is a literal module, so it has members and such that we need
+    // to resolve. First we do refinement transformation. Then we construct the signature
+    // of the module. This is the public, externally visible signature. Then we add in
+    // everything that the system defines, as well as any "import" (i.e. "opened" modules)
+    // directives (currently not supported, but this is where we would do it.) This signature,
```

Comment:
If I'm parsing this correctly, does it mean to say that `import opened` isn't supported? I think that's obsolete, if so.